### ISSUE TITLE
Fix SearchBox tests for `aria-expanded`

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/mima#414]
## Problem
As we now toggle `aria-expanded` with the presence of any autocomplete suggestions, the `SearchBox.test.js` fails to pass on this behaviour unless we mock Google Maps Autocomplete Service API to return a mock list of suggestions.

## Solution
Google Maps Autocomplete Service API to return a mock list of suggestions.
